common/Tests/Framework/src/AITestClass.ts

```diff
@@ -683,6 +731,32 @@ export class AITestClass {
         }
     }
 
+    protected _assertEventsRemoved() {
```

Comment:
This is the current set of objects that will be validated -- these will grow as we find plugins that are adding events.